index.html

```diff
@@ -12,9 +12,9 @@
 		
 	</head>
 	<body>
-		<div id="menuOverlay" class="menu_overlay">
+		<div class="menu_overlay">
 			<div class="menu_body">
-				<div id="menu_close_btn" class="menu_close_btn">
+				<div class="menu_close_btn">
```

Comment:
<button>

Reply:
Done.

Reply:
I don't see the changes. Look for all such places and use button for them

Reply:
Done.

Reply:
are you kidding?
There are no changes in it!
This is your last vesrion of code
![image](https://user-images.githubusercontent.com/37267977/217881091-24de2d43-a9ee-4445-ad57-1e305e2e1bb1.png)

Reply:
Done.